app/views/api/v1/teams/index.json.builder

```diff
@@ -0,0 +1 @@
+json.teams @teams, :id, :name, :color
```

Comment:
this file is named wrong - the extension should be `.jbuilder`

Reply:
check